src/OAuth2/ResponseType/JwtAccessToken.php

```diff
@@ -127,5 +127,15 @@ protected function createPayload($client_id, $user_id, $scope = null)
             'token_type' => $this->config['token_type'],
             'scope'      => $scope
         );
+        
+        if (isset($this->config['jwt_extra_payload']) && is_callable($this->config['jwt_extra_payload'])) {
+            $func = $this->config['jwt_extra_payload'];
+            $extra = $func($client_id, $user_id, $scope);
+            if (is_array($extra)) {
+                $payload = array_merge($extra, $payload);
```

Comment:
Do you mean that callable must be defined like this? (pay attention that `$payload` passed by reference)
`function(&$payload, $client_id, $user_id, $scope) {}`

I think that standard JWT claims should be protected from external changes. That's why I put `$payload` last in `array_merge()`